media-libs/libshumate/libshumate-1.0.1.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+VALA_USE_DEPEND="vapigen"
+
+inherit gnome.org meson vala
+
+DESCRIPTION="Shumate is a GTK toolkit providing widgets for embedded maps."
+HOMEPAGE="https://wiki.gnome.org/Projects/libshumate"
+
+SLOT="0"
+LICENSE="LGPL-2.1+"
+KEYWORDS="~amd64"
+RESTRICT="test"
```

Comment:
Why?

Reply:
I doesn't have test so I thought I should restrict them? I removed it for now, but maybe you can give some information when to use this.

Reply:
Ah, okay. Yeah, we just use `RESTRICT="test"` for cases where we don't want to run the tests because they're known to fail, or we can't run them for some reason. If the package doesn't have tests, we don't want to restrict them, because if tests are added in a subsequent version we might not notice.